libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/CiMitUtilityService.java

```diff
@@ -43,6 +43,9 @@ public Optional<JourneyResponse> getCiMitigationJourneyStep(ContraIndicators con
         var cimitConfig = configService.getCimitConfig();
         for (var ci : contraIndicators.getContraIndicatorsMap().values()) {
             if (isCiMitigatable(ci) && !isBreachingCiThresholdIfMitigated(ci, contraIndicators)) {
+                if (hasMitigatedContraIndicator(contraIndicators).isPresent()) {
```

Comment:
Maybe stick a comment in here to explain what this is for, it might be hard to work out for future us.